* fix sound/subs issue while playing MPEG-TS streams (broken aid/sid due
  to the use of alang param) : tested with DVB, RTSP Freebox and TS file.

diff -r 2006b012d73e libmpdemux/demux_ts.c
--- a/libmpdemux/demux_ts.c	Sun Apr 30 16:54:44 2006 +0200
+++ b/libmpdemux/demux_ts.c	Sun Apr 30 16:54:45 2006 +0200
@@ -978,20 +978,8 @@ static demuxer_t *demux_open_ts(demuxer_
 	params.prog = ts_prog;
 	params.probe = ts_probe;
 
-	if(dvdsub_lang != NULL)
-	{
-		strncpy(params.slang, dvdsub_lang, 3);
-		params.slang[3] = 0;
-	}
-	else
 		memset(params.slang, 0, 4);
 
-	if(audio_lang != NULL)
-	{
-		strncpy(params.alang, audio_lang, 3);
-		params.alang[3] = 0;
-	}
-	else
 		memset(params.alang, 0, 4);
 
 	start_pos = ts_detect_streams(demuxer, &params);
